applications/ShapeOptimizationApplication/custom_utilities/geometry_utilities.h

```diff
@@ -269,6 +269,17 @@ class GeometryUtilities
         KRATOS_CATCH("");
     }
 
+
+    template <class TContainerType>
+    double CalculateArea(TContainerType& rContainer)
```

Comment:
CalculateArea seems a strange name to me, given that it calculates the perimeter (length-unit) of a 2D shape.

I am not working with 2D elements/conditions often, but would it work if you call "Length()" instead of "Area()" for each condition?

Reply:
It will, for line_2d_2.h Area() returns Length():

https://github.com/KratosMultiphysics/Kratos/blob/46efa805e9a77c3fa21da64ccb8c77522d2af633/kratos/geometries/line_2d_2.h#L385-L388

The good thing about using Area() is that it will work as well for triangles (in the 3d case with wings).  So the same call can be used for 3D as well.

In any case, since Im not adding any 3D support I will rename the methods to Length() and CalculateLength() and when adding 3D support this can be changed if needed.